pandas/tests/io/test_excel.py

```diff
@@ -1212,6 +1212,17 @@ class and any subclasses, on account of the `autouse=True`
 class TestExcelWriter(_WriterBase):
     # Base class for test cases to run with different Excel writers.
 
+    # def test_excel_sheet_size(self):
+    #     assert False
```

Comment:
Not sure what this is here for but it's going to block the rest of the test so probably want to remove

Reply:
mhm. was using this to ensure i was actually py-testing the correct suite of tests. will remove.